puppet/modules/profiles/manifests/web.pp

```diff
@@ -37,6 +37,10 @@
   }
   contain web
 
+  class { 'web::vhost::archivedeb':
+  }
+  contain web::vhost::archivedeb
```

Comment:
If there are no parameters this is redundant:
```suggestion
  contain web::vhost::archivedeb
```

`contain` behaves like `include` but also makes sure that chaining works so if you have `Class['profiles::web'] -> Class['myclass']` you can be sure that there's `Class['web::vhost::archivedeb'] -> Class['myclas']`. With just `include` it "floats" and can show up anywhere in the dependency graph.